ci/test-flow-visibility.sh

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+CLICKHOUSE_URL="tcp://clickhouse-clickhouse.flow-visibility:9000"
+FAILURE=false
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+GIT_CHECKOUT_DIR=${THIS_DIR}/..
+
+_usage="Usage: $0 [--kubeconfig <KubeconfigSavePath>]
+
+Setup Grafana, Clickhouse and Antrea Flow Aggregator to validate flow-aggregator.yml and flow-visibility.yml.
+
+    --kubeconfig    Path to kubeconfig.
+
+"
+
+echoerr() {
+  >&2 echo "$@"
+}
+
+print_usage() {
+  echoerr "$_usage"
+}
+
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+case $key in
+    --kubeconfig)
+    export KUBECONFIG="$2"
+    echo "kube"
+    shift 2
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+start_antrea() {
+  echo "=== Starting Antrea ==="
+  kubectl apply -f ${GIT_CHECKOUT_DIR}/build/yamls/antrea.yml
+  kubectl rollout status --timeout=5m deployment/coredns -n kube-system
+  kubectl rollout status --timeout=5m deployment.apps/antrea-controller -n kube-system
+  kubectl rollout status --timeout=5m daemonset/antrea-agent -n kube-system
+}
+
+setup_flow_visibility() {
+  echo "=== Starting Flow Visibility ==="
+  # install Clickhouse operator
+  kubectl apply -f https://raw.githubusercontent.com/Altinity/clickhouse-operator/master/deploy/operator/clickhouse-operator-install-bundle.yaml
+  kubectl apply -f ${GIT_CHECKOUT_DIR}/build/yamls/flow-visibility.yaml
+  echo "=== Waiting for Clickhouse and Grafana to be ready ==="
```

Comment:
nits: Clickhouse -> ClickHouse, same for below

Reply:
Thanks, addressed.